windows/src/engine/keyman32/K32_load.cpp

```diff
@@ -75,12 +75,55 @@ BOOL GetKeyboardFileName(LPSTR kbname, LPSTR buf, int nbuf)
   return n;
 }
 
+BOOL LoadlpKeyboardCore(int i)
+{
+  SendDebugMessageFormat(0, sdmAIDefault, 0, "LoadlpKeyboardCore: Enter ---");
+  PKEYMAN64THREADDATA _td = ThreadGlobals();
+  if (!_td) return FALSE;
+  if (_td->lpKeyboards[i].coreKeyboard) return TRUE;
+  if (_td->lpActiveKeyboard == &_td->lpKeyboards[i]) _td->lpActiveKeyboard = NULL;  // I822 TSF not working
+
+  if (_td->lpKeyboards[i].lpActiveKBState) {
+    km_kbp_state_dispose(_td->lpKeyboards[i].lpActiveKBState);
+    _td->lpKeyboards[i].lpActiveKBState = NULL;
+  }
+
+  char buf[256];
+  if (!GetKeyboardFileName(_td->lpKeyboards[i].Name, buf, 255)) return FALSE;
+  PWCHAR keyboardPath = strtowstr(buf);
+  km_kbp_status_codes err_code = (km_kbp_status_codes)km_kbp_keyboard_load(keyboardPath, &_td->lpKeyboards[i].coreKeyboard);
+  if (err_code != KM_KBP_STATUS_OK) {
+    delete keyboardPath;
+    return FALSE;
+  }
+  delete keyboardPath;
```

Comment:
```suggestion
  delete keyboardPath;
  if (err_code != KM_KBP_STATUS_OK) {\
    SendDebugMessageFormat(...); // TODO: log err_code
    return FALSE;
  }
```

In all failure paths, can we log `err_code` with `SendDebugMessageFormat` so we don't lose it? (Masking error details is a pet peeve of mine 😬)